app/components/gh-member-settings-form.js

```diff
@@ -33,12 +33,14 @@ export default Component.extend({
         let subscriptions = this.member.get('stripe');
         if (subscriptions && subscriptions.length > 0) {
             return subscriptions.map((subscription) => {
+                const statusLabel = subscription.status === 'past_due' ? 'Past due' : subscription.status;
```

Comment:
Does this need to change `'active'` to `'Active'` and that? Or is that done in CSS or something?

Reply:
That's done in the CSS already, basically capital-cases the status value